grader/reject.go

```diff
@@ -125,3 +135,28 @@ func checkSubmissionLimitWindow(window *model.SubmittionLimitWindow,
 
     return nil, nil;
 }
+
+func checkLateAcknowledgment(assignment *model.Assignment, lateAcknowledgment bool) (RejectReason, error) {
```

Comment:
I don't see a need for the error return value.

Reply:
eeca6fb42f0cd2fcc0c243665a060acbb24e7edf